python/mqt/qcec/compilation_flow_profiles.py

```diff
@@ -21,7 +21,7 @@
 
 if TYPE_CHECKING:
     from numpy.typing import NDArray
-    from qiskit.circuit import QuantumCircuit
+    from qiskit import QuantumCircuit
```

Comment:
I am pretty sure that `qiskit.circuit` is the "right" import here?
I remember the current one causing some problems with sphinx and autoAPI. Might be misremembering though